setup.py

```diff
@@ -31,6 +31,8 @@
     package_data={"stripe": ["data/ca-certificates.crt"]},
     zip_safe=False,
     install_requires=[
+        'typing_extensions <= 4.2.0; python_version < "3.7"',
```

Comment:
```suggestion
        'typing_extensions <= 4.8.0; python_version < "3.7"',
```

Reply:
I don't understand why you are suggesting this? 4.2.0 dropped support for 3.6